src/lsp/ext.rs

```diff
@@ -51,6 +53,28 @@ impl Notification for CorelibVersionMismatch {
     const METHOD: &'static str = "cairo/corelib-version-mismatch";
 }
 
+/// Collects versions of LS and it's dependencies.
+#[derive(Debug)]
+pub struct LSVersions;
```

Comment:
```suggestion
pub struct ToolchainInfo;
```

to me it sounds like we might want to stick more information in this struct in the future and this name looks more general to me. wdyt?